scripts/test.sh

```diff
@@ -221,8 +228,14 @@ if [[ -z "$(command -v nvm)" ]]; then
   set -e
 fi
 if [[ "$(command -v nvm)" ]]; then
-  nvm install 6.11.3
+  nvm install $NODE_VERSION
 fi
+set_nvm_default() {
+  if [[ "$(command -v nvm)" ]]; then
+    nvm_old_default="$(nvm alias default --no-colors | cut -d ' ' -f 3)"
+    nvm alias default $(nvm current)
```

Comment:
I don't like that we are modifying a system global setting on the developers machine here - that's a bit too unexpected in my opinion.

Reply:
I guess I can probably move this to the Jenkinsfile and just make check-environment.sh verify that the default it set to something which will work.